Project.toml

```
@@ -1,7 +1,7 @@
 name = "Onda"
 uuid = "e853f5be-6863-11e9-128d-476edb89bfb5"
 authors = ["Beacon Biosignals, Inc."]
-version = "0.15.7"
+version = "0.15.8"
```

if we do go with this approach (still no convinced), it would definitely be breaking.
```suggestion
version = "0.16.0"
```

---

I've revised the change to be non-breaking.